Extension/package.json

```diff
@@ -43,6 +43,33 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "taskDefinitions": [
+      {
+        "type": "cppbuild",
+        "required": [
+          "command",
+          "label"
+        ],
+        "properties": {
+          "label": {
+            "type": "string",
+            "description": "Task's name."
```

Comment:
We need to localize these

Reply:
Description texts:
`name` - "The name of the task."
`command` - "The path to either a compiler or script that performs compilation."
`args` - "Additional arguments to pass to the compiler or compilation script."